docs/curation-review-tut.md

```diff
@@ -0,0 +1,417 @@
+    disease_ontology_term_id: PATO:0000461                                  #MONDO disease term or PATO:0000461
+    sex: male                                                               #male, female, mixed, unknown, or other
+    ethnicity_ontology_term_id: na                                          #HANCESTRO term
+    development_stage_ontology_term_id: HsapDv:0000160                      #HsapDv term
+    cell_type_ontology_term_id:
+      cell_label:                                                           #this is column name in your obs dataframe (the field that specifies cell type)
+        acinar: CL:0002064                                                  #mapping between cell types (specified in anndata object) and cl ontology id
+        alpha: CL:0000171
+        delta: CL:0000173
+        endothelial: CL:0000115
+        epsilon: CL:0005019
+
+```
+
+
+<br/>
+
+In the config file snippet above, the 0 level indentation specifies that we are modifying `obs`. At the first level of indentation, we start adding schema fields to `obs`. If the first level indentation fields ar follow by an ontology value (i.e. `tissue_ontology_term_id: UBERON:0000006`), then a column called `tissue_ontology_term_id` will created in the obs dataframe, and all of its values will be `UBERON:0000006`. Additionally, another column named `tissue` will be created with the corresponding UBERON term (`islet of Langerhans`).
+
+On other hand, if the schema field is followed by lines with have further indentation, then the next line in the config file specify a column whose values should be mapped to ontology terms (with these mappings specified by the subsequent lines). Two new columns will be added to the `obs` data frame `cell_type_ontology_term_id` and `cell_type`.
+
+<br/>
+
+@TODO - the fixup gene symbols section
+
+</details>
+
+<br/>
+
+
+#### `cellxgene schema apply`
+ 
+<details>
+ 
+<summary> applying the schema </summary>
+
+<br/>
+
+In order to use the `cellxgene schema apply` command, you will need to pass the following required arguments:
+- `--source-h5ad` your original anndata file
+- `--remix-config` the `config.yaml` file that we specified above 
+- `--output-filename` the name of the resulting anndata that is consistent with the cellxgene schema
+
+<br/>
+
+The next step will be to validate the resulting object.
+
+</details>
+
+<br/>
+
+#### `cellxgene schema validate`
+
+<details>
+ 
+<summary> validating the schema</summary>
+
+<br/>
+
+In order to validate the remixed object, needs to simply run `cellxgene schema validate remixed_anndata.h5ad`. If there has been no terminal output from the function, then your object has been validated successfuly and is ready for upload!
+
+</details>
+
+<br/>
+
+### Test `h5ad` in local version of cellxgene
+
+## Uploading data to the cellxgene data portal
+
+Introduce collection view, dataset view, and explorer view
```

Comment:
We're moving towards _My Collection_ and _A Collection_ for more clarity.

Reply:
amended this terminology